modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -9,47 +9,73 @@
 
 The following `ImageSetConfiguration` file examples show the configuration for various mirroring use cases.
 
+// Moved to first; unchanged
 [discrete]
-[id="oc-mirror-image-set-examples-helm_{context}"]
-== Use case: Including arbitrary images and helm charts
+[id="oc-mirror-image-set-examples-shortest-upgrade-path_{context}"]
+== Use case: Including the shortest {product-title} upgrade path
 
-The following `ImageSetConfiguration` file uses a registry storage backend and includes helm charts and an additional Red Hat Universal Base Image (UBI).
+The following `ImageSetConfiguration` file uses a local storage backend and includes all {product-title} versions along the shortest upgrade path from the minimum version of `4.11.37` to the maximum version of `4.12.15`.
```

Comment:
when use the shortpath , will not contain all the version , only contain the minimum and maximum , I think the description here is a little confused .

Reply:
I thought that sometimes there were required stops along the path to upgrading, that might require versions in between the minimum an the maximum versions specified. @sherine-k or @aguidirh Can you confirm, and see if we need to make any adjustments to the wording here?

Reply:
I'm not a big expert on cincinatti graphs, but from the [tests](https://github.com/openshift/oc-mirror/blob/2445f2cf88ee364085dfdb618994c2f99316ed67/pkg/cli/mirror/release_test.go#L143), I could see that shortestPath is not necessarily always just the 2 versions min and max.

Reply:
@bergerhoffer  @sherine-k Got you , thanks .